.buildkite/publish-react-native-ios-artifacts.sh

```diff
@@ -1,16 +1,22 @@
 #!/bin/bash -eu
 
-cd ./ios-xcframework
+# Make Homebrew run a bit faster
+export HOMEBREW_NO_AUTO_UPDATE=1
+export HOMEBREW_NO_INSTALL_CLEANUP=1
```

Comment:
Having these exports here rather than inline is mostly cosmetic. I did it because I was `brew install`ing another tool at some point in development, but that's no longer the case.

I considered using the `env` node in the step definition in `pipeline.yml` but I wasn't sure whether we propagate those values to the scripts we run. Besides, given the code that uses these var is in this file, it make sense to me to leave them here.

Reply:
Also notice it's been discussed (internal ref p1682611494399739-slack-C020Q0Z579V ) adding these to our VM setup upstream. When that will happen, we'll be able to remove these from here.

Reply:
`HOMEBREW_NO_AUTO_UPDATE` is already included. `HOMEBREW_NO_INSTALL_CLEANUP` is not. But I don't think we need the second one? I don't imagine "cleaning up old formula" would have an big impact on the performance.

> HOMEBREW_NO_INSTALL_CLEANUP
>
> If set, brew install, brew upgrade and brew reinstall will never automatically cleanup installed/upgraded/reinstalled formulae or all formulae every HOMEBREW_CLEANUP_PERIODIC_FULL_DAYS days. Alternatively, HOMEBREW_NO_CLEANUP_FORMULAE allows specifying specific formulae to not clean up.